packages/pigeon/lib/dart_generator.dart

```diff
@@ -349,7 +352,7 @@ $resultAt != null
         indent.newln();
       }
       indent.write(
-          'static void setup(${api.name}? api, {BinaryMessenger? binaryMessenger}) ');
+          "static void setup(${api.name}? api, {BinaryMessenger? binaryMessenger, String messageChannelSuffix = '',}) ");
```

Comment:
I don't suppose I could convince you to make this a new method called `setUp`, and the old one deprecated and delegate to the new version? 😬 

(I know it's not really related, but you're touching it so I had to try...)